front/src/components/appBar.js

```diff
@@ -36,7 +36,7 @@ const toolBarStyle = {
  * classement) and the session informations (number of exercices)
  * @return {Component}
  */
-function ResponsiveAppBar({user, session}) {
+function ResponsiveAppBar({user, otherInfos, session, logoutUserStudent}) {
```

Comment:
`otherInfos` is not very precise let think about a more precise props name

Reply:
It's temporary, because actually a student doesn't have his score and his rank stored in the db. But yeah, I rename otherInfos as fixedUserInfos.